app/src/main/java/org/oppia/app/activity/InputInteractionViewTestActivity.kt

```diff
@@ -4,10 +4,12 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import org.oppia.app.R
 import org.oppia.app.customview.interaction.NumericInputInteractionView
+import org.oppia.app.customview.interaction.TextInputInteractionView
+import org.oppia.app.customview.interaction.FractionInputInteractionView
 
 /**
  * This is a dummy activity to test input interaction views.
- * It contains [NumericInputInteractionView] .
+ * It contains [NumericInputInteractionView], [TextInputInteractionView] and [FractionInputInteractionView].
  */
 class InputInteractionViewTestActivity : AppCompatActivity() {
```

Comment:
Move this file in testing from activity package

Reply:
Resolved